openfl-workspace/tf_2dunet/src/tf_2dunet.py

```diff
@@ -173,6 +173,8 @@ def define_model(input_tensor,
         **kwargs: Additional parameters to pass to the function
 
     """
+    if not dropout_at:
```

Comment:
It's better use ` if dropout_at is None:` for such cases. Otherwise, you will set the default value not only for those cases when the parameter was not passed but also when `bool(parameter) is True` .